Kyle_Marienthal/DJANGO/belt_reviewer/apps/log_reg_app/models.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+import bcrypt
+
+# Create your models here.
+class UserManager(models.Manager):
+    def validate_registration(self, form_data):
+        print '*****youre in the validate_registration method*****'
+
+        #form_data here is request.POST['whatever form info you need']
+        errors = []
+        if len(form_data['name']) == 0:
+            errors.append('Name can not be blank')
+        if len(form_data['alias']) == 0:
+            errors.append('Alias can not be blank')
+        if len(form_data['email']) == 0:
+            errors.append('Email can not be blank')
+        # # make sure theyre not a user in the DB already
+        # user = User.objects.filter(email=form_data['email']).first()
```

Comment:
If you wanted to get this to work. You could try self.filter instead of User.objects.filter